tests/spread/general/entrypoint-service/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: container entrypoint-service test
+
+execute: |
+  run_rockcraft pack
+
+  test -f entrypoint-service-test_latest_amd64.rock
+
+  # test container execution
+  sudo /snap/rockcraft/current/bin/skopeo --insecure-policy copy oci-archive:entrypoint-service-test_latest_amd64.rock docker-daemon:entrypoint-service-test:latest
+  rm entrypoint-service-test_latest*.rock  
+  docker images entrypoint-service-test:latest
+  id=$(docker run -d entrypoint-service-test)
+  grep_docker_log "$id" "foo"
+  docker rm -f "$id"
+  id=$(docker run -d entrypoint-service-test bar)
+  grep_docker_log "$id" "bar"
+  docker rm -f "$id"
```

Comment:
you're missing the `docker inspect` test we discussed offline, i.e.:

`docker inspect entrypoint-service-test:latest -f '{{json .Config.Entrypoint}}' == '["/bin/pebble","enter","--verbose", "--args", "test-service"]'`

and

`docker inspect entrypoint-service-test:latest -f '{{json .Config.Cmd}}' == '["foo"]'`